pkgs/development/libraries/science/astronomy/indilib/indi-3rdparty.nix

```
@@ -32,23 +35,23 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     indilib libnova curl cfitsio libusb1 zlib boost gsl gpsd
-    libjpeg libgphoto2 libraw libftdi1 libdc1394 ffmpeg fftw
+    libjpeg libgphoto2 libraw.dev libraw.lib libftdi1 libdc1394 ffmpeg fftw
+    limesuite libraw1394 udev
```

The packages build the same for me without these. Maybe I overlook something, but `libraw1394` is not referenced in the transitive closure of `indi-full`, so probably safe to remove.

```suggestion
    limesuite
```